src/t8_geometry/t8_geometry_implementations/t8_geometry_analytic.hxx

```diff
@@ -0,0 +1,164 @@
+/*
```

Comment:
Isnt this just a C interface to the geometry?
Anyone using C++ can inherit the geometry class to define their own geometry.
No need to be using this class and hassling with function pointers.